.gitignore

```diff
@@ -3,4 +3,4 @@ taskcat_outputs/
 .taskcat/
 *.zip
 .idea
-index.html
\ No newline at end of file
+index.html
```

Comment:
Please remove this line, it can cause issues with publishing the deployment guide if this .gitignore is accidentally merged into `doc-edits` branch. You can ignore the file for yourself in a global .gitignore file.

Reply:
updated